doc/changes_from_dnf4.7.rst

```diff
@@ -237,7 +237,7 @@ Changes to individual commands
 
 ``makecache``
   * Metadata is now stored in different directories, see the ``cachedir`` configuration option :ref:`changes <cachedir_option_conf_changes_ref-label>` for more details.
-  * The ``--timer`` option has been dropped in favor of the systemd ``OnUnitInactiveSec`` setting in ``dnf5-makecache.timer`` and the ``ConditionACPower`` setting in ``dnf5-makecache.service``.
+  * The ``--timer`` option has been dropped in favor of the systemd ``OnUnitInactiveSec`` setting in ``dnf-makecache.timer`` and the ``ConditionACPower`` setting in ``dnf-makecache.service`` (these files are named ``dnf5-makecache.timer`` and ``dnf5-makecache.service`` on some systems).
```

Comment:
This rename in the documentation should also subject of the %{with dnf5_obsoletes_dnf} condition.

Reply:
Done.